proposals/lip-0074.md

```diff
@@ -454,8 +453,8 @@ configSchema = {
 |   **Name**      |  **Type**   |   **Modifiable**  |**Initial Mainchain Value**|**Description**         |
 |-----------------|-------------|-------------------|----------------------------------|------------------------|
 |`tokenIdTreasury`|   `bytes`   |     No            | `TOKEN_ID_LSK`                   |Token ID of the tokens stored in the treasury and handled by funding proposals.|
-|`voteDuration`   | `uint32`    |       No          |   240000                         |Length of the vote period in blocks. As clarified in [Rationale](#voting-on-a-proposal), `voteDuration <= LOCKING_PERIOD_STAKING`, where `LOCKING_PERIOD_STAKING` is defined in [LIP 0057][posModule].|
-|`quorumDuration` | `uint32`    |       No          |   120000                         |Length of the quorum period in blocks. After this period the quorum is checked.|
+|`voteDuration`   | `uint32`    |       No          |   `241,920 = 28 * 24 * 3600 // BLOCK_TIME`                         |Length of the vote period in blocks. As clarified in [Rationale](#voting-on-a-proposal), `voteDuration <= LOCKING_PERIOD_STAKING`, where `LOCKING_PERIOD_STAKING` is defined in [LIP 0057][posModule].|
+|`quorumDuration` | `uint32`    |       Yes          |   `120,960 = 14 * 24 * 3600 // BLOCK_TIME`                         |Length of the quorum period in blocks. After this period the quorum is checked.|
```

Comment:
If the goal is to configure these durations based on time, not on number of block passed, it would be better to simply define them as `28 * 24 * 3600 // BLOCK_TIME` and `14 * 24 * 3600 // BLOCK_TIME` and remove the calculated values `241,920` and `120,960`.

The current specification is misleading IMO, because it assumes that the `BLOCK_TIME` is `10`, while in fact it is a configurable value.

Reply:
I think the current version is fine. The LIP suggests initial values for the Mainchain, so it is fair to assume that `BLOCK_TIME = 10`. Moreover I mention this parameter explicitly, so if in the future we decide to change `BLOCK_TIME` value, a simple search will also indicate that the Governance module parameters should be updated accordingly.